x-pack/plugins/index_management/public/application/services/documentation.ts

```diff
@@ -63,157 +144,154 @@ class DocumentationService {
     }
     return `${this.esDocsBase}${typeDefinition.documentation[docType]}`;
   };
-
   public getMappingTypesLink() {
-    return `${this.esDocsBase}/mapping-types.html`;
+    return this.mappingFieldDataTypes;
   }
-
   public getDynamicMappingLink() {
-    return `${this.esDocsBase}/dynamic-field-mapping.html`;
+    return this.mappingDynamicFields;
   }
-
   public getPercolatorQueryLink() {
-    return `${this.esDocsBase}/query-dsl-percolate-query.html`;
+    return this.percolate;
   }
 
   public getRankFeatureQueryLink() {
-    return `${this.esDocsBase}/rank-feature.html`;
+    return this.mappingRankFeatureFields;
   }
 
   public getMetaFieldLink() {
-    return `${this.esDocsBase}/mapping-meta-field.html`;
+    return this.mappingMetaFields;
   }
 
   public getDynamicTemplatesLink() {
-    return `${this.esDocsBase}/dynamic-templates.html`;
+    return this.mappingDynamicTemplates;
   }
 
   public getMappingSourceFieldLink() {
-    return `${this.esDocsBase}/mapping-source-field.html`;
+    return this.mappingSourceFields;
   }
 
   public getDisablingMappingSourceFieldLink() {
-    return `${this.esDocsBase}/mapping-source-field.html#disable-source-field`;
+    return this.mappingSourceFieldsDisable;
   }
 
   public getNullValueLink() {
-    return `${this.esDocsBase}/null-value.html`;
+    return this.mappingNullValue;
   }
 
   public getTermVectorLink() {
-    return `${this.esDocsBase}/term-vector.html`;
+    return this.mappingTermVector;
   }
 
   public getStoreLink() {
-    return `${this.esDocsBase}/mapping-store.html`;
+    return this.mappingStore;
   }
 
   public getSimilarityLink() {
-    return `${this.esDocsBase}/similarity.html`;
+    return this.mappingSimilarity;
   }
 
   public getNormsLink() {
-    return `${this.esDocsBase}/norms.html`;
+    return this.mappingNorms;
   }
 
   public getIndexLink() {
-    return `${this.esDocsBase}/mapping-index.html`;
+    return this.mappingIndex;
   }
 
   public getIgnoreMalformedLink() {
-    return `${this.esDocsBase}/ignore-malformed.html`;
+    return this.mappingIgnoreMalformed;
   }
 
   public getMetaLink() {
-    return `${this.esDocsBase}/mapping-field-meta.html`;
+    return this.mappingMeta;
   }
 
   public getFormatLink() {
-    return `${this.esDocsBase}/mapping-date-format.html`;
+    return this.mappingFormat;
   }
 
   public getEagerGlobalOrdinalsLink() {
-    return `${this.esDocsBase}/eager-global-ordinals.html`;
+    return this.mappingEagerGlobalOrdinals;
   }
 
   public getDocValuesLink() {
-    return `${this.esDocsBase}/doc-values.html`;
+    return this.mappingDocValues;
   }
 
   public getCopyToLink() {
-    return `${this.esDocsBase}/copy-to.html`;
+    return this.mappingCopyTo;
   }
 
   public getCoerceLink() {
-    return `${this.esDocsBase}/coerce.html`;
+    return this.mappingCoerce;
   }
 
   public getBoostLink() {
-    return `${this.esDocsBase}/mapping-boost.html`;
+    return this.mappingParameters;
```

Comment:
I'm not sure if this was intentional or a mistake, but I believe the doc link should be https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-boost.html. However, the `boost` parameter is deprecated in `8.0` so the doc link is only valid on `7.x`.

Reply:
Yes, that was intentionally changed since the mapping-boost.html page 404s in master.  If there's a better page to link to in that branch, just let me know!